fix: standalone ajv schema (#461)
fastify/fast-json-stringify: README.md

### Standalone Mode

The standalone mode is used to compile the code that can be directly run by `node`

> itself. You need to install `ajv`, `fast-uri` and `ajv-formats` in order to let 
> the standalone code works.

Reviewer: ```suggestion
itself. You need to install `ajv`, `fast-uri` and `ajv-formats` for
the standalone code to work.
```
